tests/test_numpy_array.py

```diff
@@ -481,6 +481,9 @@ def test_index_using_ellipsis():
     a = m.index_using_ellipsis(np.zeros((5, 6, 7)))
     assert a.shape == (6,)
 
+def test_format_descriptors_for_const_types():
```

Comment:
For completeness, could you please add corresponding non-const tests, even if redundant (I didn't look if that's covered elsewhere already). It's usually good to exercise closely related contrasting functionality together, especially if it's super easy, as in this case.